src/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>Movies App</title>
+</head>
+
+<body>
+
+    <header>
+        <div class="container">
+
+            <nav>
+                <div class="logo">
+                    <h1> Bahati & Antoine </h1>
+                </div>
+
+                <ul>
+                    <li>
+                        <span><a href="index.html" class="active">Browse</a>
+                        </span>
+                    </li>
+                    <li>
+                        <span><a href="#">Movies</a>
+                        </span>
+                    </li>
```

Comment:
- [ ] Kindly add a counter for the number of movies on your website as shown on the wireframe. I have attached an image for your reference.

![count](https://github.com/ANTOINE1128/javascript-capstone/assets/53124548/e6f30f27-ceb8-4965-b750-cdd4ef75bcb7)